monkey/infection_monkey/network/fingerprinter_manager.py

```diff
@@ -0,0 +1,13 @@
+import logging
+from infection_monkey.utils.load_plugins import get_instances
+from infection_monkey.network.HostFinger import HostFinger
+
+LOG = logging.getLogger(__name__)
+
+
+def get_fingerprint_instances():
```

Comment:
Use typing also `def get_fingerprint_instances() -> List[HostFinger]:`

Reply:
Done. With sequence and not List